packages/manager/src/features/CloudPulse/Alerts/AlertsListing/AlertListing.tsx

```diff
@@ -1,88 +1,228 @@
-            <TableRowError
-              colSpan={7}
-              message={'Error in fetching the alerts.'}
-            />
-          )}
-          {isLoading && <TableRowLoading columns={7} />}
-          {alerts?.map((alert) => (
-            <AlertTableRow
-              handlers={{
-                handleDetails: () => handleDetails(alert),
-              }}
-              alert={alert}
-              key={alert.id}
-            />
-          ))}
-        </TableBody>
-      </Table>
-    </Grid>
+    <Stack spacing={2}>
+      <Box
+        alignItems={{ lg: 'flex-end', md: 'flex-start' }}
+        display="flex"
+        flexDirection={{ lg: 'row', md: 'column', sm: 'column', xs: 'column' }}
+        gap={3}
+        justifyContent="space-between"
+      >
+        <Box
+          flexDirection={{
+            lg: 'row',
+            md: 'column',
+            sm: 'column',
+            xs: 'column',
+          }}
+          display="flex"
+          gap={2}
+        >
+          <DebouncedSearchTextField
+            sx={{
+              width: { md: '300px', sm: '500px', xs: '300px' },
+            }}
+            data-qa-filter="alert-search"
+            label=""
+            noMarginTop
+            onSearch={setSearchText}
+            placeholder="Search for Alerts"
+            value={searchText}
+          />
+          <Autocomplete
+            errorText={
+              serviceTypesError ? 'An error in fetching the services.' : ''
+            }
+            onChange={(_, selected) => {
+              setServiceFilters(selected);
+            }}
+            sx={{
+              width: { md: '300px', sm: '500px', xs: '300px' },
+            }}
+            autoHighlight
+            data-qa-filter="alert-service-filter"
+            data-testid="alert-service-filter"
+            label={''}
+            limitTags={2}
+            loading={serviceTypesLoading}
+            multiple
+            noMarginTop
+            options={getServicesList}
+            placeholder={serviceFilters.length > 0 ? '' : 'Select a Service'}
+            value={serviceFilters}
+          />
+          <Autocomplete
+            onChange={(_, selected) => {
+              setStatusFilters(selected);
+            }}
+            sx={{
+              width: { md: '300px', sm: '500px', xs: '300px' },
+            }}
+            autoHighlight
+            data-qa-filter="alert-status-filter"
+            data-testid="alert-status-filter"
+            label={''}
+            multiple
+            noMarginTop
+            options={alertStatusOptions}
+            placeholder={statusFilters.length > 0 ? '' : 'Select a Status'}
+            value={statusFilters}
+          />
+        </Box>
+        <Button
+          onClick={() => {
+            history.push(`${url}/create`);
+          }}
+          sx={{
+            height: '34px',
+            paddingBottom: 0,
+            paddingTop: 0,
+            width: { md: '150px', xs: '200px' },
```

Comment:
```suggestion
            height: '34px',
            paddingBottom: 0,
            paddingTop: 0,
            whiteSpace: 'nowrap',
            width: { md: '150px', xs: '200px' },
```

prevent the button text from wrapping as the viewport shrinks